### ISSUE TITLE
error using omni-configure with Kentucky Emulab

### ISSUE THREAD

**Issue by seledwards** [GENI-NSF/geni-tools#453]
omni-configure may not handle certs from Kentucky Emulab.

Hussam suggests the following fix:
```
def getPGConfig(opts, public_key_list, cert) :

    (user, user_urn) = getUserInfo(cert)

    if user_urn.find('uky.emulab.net') != -1 :
        sa = 'https://www.uky.emulab.net:12369/protogeni/xmlrpc/sa'
       elseif user_urn.find('emulab.net'): != -1 :
            sa = 'https://www.emulab.net:12369/protogeni/xmlrpc/sa'
    elseif user_urn.find('pgeni.gpolab.bbn.com') != -1 :
            sa = 'https://www.pgeni.gpolab.bbn.com:12369/protogeni/xmlrpc/sa'
    elseif user_urn.find('loni.org') != -1 :
            sa = 'https://cron.loni.org:443/protogeni/xmlrpc/sa'
    else:
            raise Exception("Creation of omni_config for users at %s is not supported. Please contact omni-users@geni.net" % user_urn.split('+')[-2])
    logger.debug("Framework is ProtoGENI, use as SA: %s", sa)

```

 _Imported from trac ticket [#453](http://trac.gpolab.bbn.com/gcf/ticket/453),  created by sedwards on 12-10-2013 at 10:53, last modified: 01-16-2014 at 13:28_
   CCing: hussam.nasir@..., gpo-expt-support@...

**Comment by ahelsing** [comment]
Updating CC --- Trac does not handle human-readable names in e-mail addresses, only a comma-separated list of addresses.

_Trac comment by chaos on 12-10-2013 at 10:58_

**Comment by innnkkki** [comment]
Fixed for gcf 2.5. 

_Trac comment by nriga (github user: innnkkki) on 01-16-2014 at 13:28_
